etc/service-bus.api.ts

```diff
@@ -0,0 +1,411 @@
+  receiveMode?: ReceiveMode;
+}
+
+// @public
+interface ReceivedMessageInfo {
+}
+
+// @public
+class ReceiveHandler {
+  // WARNING: The type "MessageReceiver" needs to be exported by the package (e.g. added to index.ts)
+  // WARNING: The type "MessageReceiver" needs to be exported by the package (e.g. added to index.ts)
+  constructor(receiver: MessageReceiver);
+  readonly address: string | undefined;
+  readonly autoComplete: boolean;
+  readonly isReceiverOpen: boolean;
+  readonly name: string;
+  readonly receiveMode: ReceiveMode;
+  stop(): Promise<void>;
+}
+
+// @public
+enum ReceiveMode {
+  peekLock = 1,
+  receiveAndDelete = 2
+}
+
+// @public
+interface RuleDescription {
+  action?: SQLExpression;
+  filter?: SQLExpression | CorrelationFilter;
+  name: string;
+}
+
+// @public
+interface ScheduleMessage {
+  message: SendableMessageInfo;
+  scheduledEnqueueTimeUtc: Date;
+}
+
+// @public (undocumented)
+interface SendableMessageInfo {
+}
+
+// @public
+interface ServiceBusConnectionStringModel {
+  // (undocumented)
+  [x: string]: any;
+  // (undocumented)
+  Endpoint: string;
+  // (undocumented)
+  EntityPath?: string;
+  // (undocumented)
+  SharedAccessKey: string;
+  // (undocumented)
+  SharedAccessKeyName: string;
+}
+
+// @public
+interface ServiceBusDeliveryAnnotations extends DeliveryAnnotations {
+  [x: string]: any;
+  last_enqueued_offset?: string;
+  last_enqueued_sequence_number?: number;
+  last_enqueued_time_utc?: number;
+  runtime_info_retrieval_time_utc?: number;
+}
+
+// @public
+class ServiceBusMessage implements ReceivedMessage {
+  // WARNING: The type "ClientEntityContext" needs to be exported by the package (e.g. added to index.ts)
+  constructor(context: ClientEntityContext, msg: AmqpMessage, delivery: Delivery);
+  readonly _amqpMessage: AmqpMessage;
+  abandon(propertiesToModify?: Dictionary<any>): Promise<void>;
+  body: any;
+  clone(): SendableMessageInfo;
+  complete(): Promise<void>;
+  contentType?: string;
+  correlationId?: string | number | Buffer;
+  deadLetter(options?: DeadLetterOptions): Promise<void>;
+  readonly deadLetterSource?: string;
+  defer(propertiesToModify?: Dictionary<any>): Promise<void>;
+  readonly delivery: Delivery;
+  readonly deliveryCount?: number;
+  readonly enqueuedSequenceNumber?: number;
+  readonly enqueuedTimeUtc?: Date;
+  readonly expiresAtUtc?: Date;
+  label?: string;
+  lockedUntilUtc?: Date;
+  readonly lockToken?: string;
+  messageId?: string | number | Buffer;
+  partitionKey?: string;
+  replyTo?: string;
+  replyToSessionId?: string;
+  scheduledEnqueueTimeUtc?: Date;
+  readonly sequenceNumber?: Long;
+  sessionId?: string;
+  timeToLive?: number;
+  to?: string;
+  userProperties?: Dictionary<any>;
+  viaPartitionKey?: string;
+}
+
+// @public
+interface ServiceBusMessageAnnotations extends MessageAnnotations {
+  // WARNING: The name "x-opt-enqueued-time" contains unsupported characters; API names should use only letters, numbers, and underscores
```

Comment:
This needs to be suppressed. That is how the property is sent on the wire. We cannot change it.

Reply:
`ServiceBusDeliveryAnnotations` and `ServiceBusMessageAnnotations` are not exposed to the user anymore. It wasnt clear how they would get used by the user. So removed until there is a need to have them back